packages/feedback-react/src/questions/CheckboxQuestion.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ChangeEventHandler } from "react";
+
+import { FieldGroup } from "@fremtind/jkl-field-group-react";
+import { Checkbox } from "@fremtind/jkl-checkbox-react";
+import { useFollowUpContext } from "../followup/followupContext";
+import { useMainQuestionContext } from "../main-question/mainQuestionContext";
+import { FeedbackOption, QuestionProps } from "../types";
+
+export const CheckboxQuestion: React.VFC<QuestionProps> = ({ label, name, options, helpLabel }) => {
+    const followupContext = useFollowUpContext();
+    const feedbackContext = useMainQuestionContext();
+    const context = followupContext || feedbackContext;
+
+    const handleChange: ChangeEventHandler<HTMLInputElement> = (event) => {
+        const { value } = event.target;
+        const matchingOption = options?.find((option) => option.value.toString() === value);
+        if (!matchingOption) return;
+
+        if (!context?.currentValue) {
+            context?.setCurrentValue([matchingOption]);
+            return;
+        }
+
+        if (Array.isArray(context?.currentValue)) {
+            const found = context.currentValue.find((option) => option === matchingOption);
+            if (found) {
+                context.setCurrentValue((oldValues) =>
+                    (oldValues as FeedbackOption[]).filter((option) => option !== found),
```

Comment:
isNotStrictlyEqual

Reply:
Hvor kommer navnet `isNotStrictlyEqual` fra? "Er ikke strengt lik" høres it som funksjonen gjør `==` i mine ører.
`isStrictlyNotEqual` er kanskje tydeligere, hvis dette er et navn vi har kontroll over.

Reply:
Ja enig, var bare noe jeg kom på i farta :D